pkg/antctl/raw/packetcapture/command.go

```diff
@@ -0,0 +1,337 @@
+// Copyright 2025 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packetcapture
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/utils/ptr"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	antrea "antrea.io/antrea/pkg/client/clientset/versioned"
+	"antrea.io/antrea/pkg/util/env"
+)
+
+var (
+	defaultTimeout          = time.Second * 60
+	maxPacketCaptureTimeout = time.Second * 300
+	Command                 *cobra.Command
+	getClients              = getConfigAndClients
+	getCopier               = getPodFileCopier
+	defaultFS               = afero.NewOsFs()
+)
+
+type options struct {
+	source    string
+	dest      string
+	nowait    bool
+	timeout   time.Duration
+	number    int32
+	flow      string
+	outputDir string
+}
+
+var option = &options{}
```

Comment:
maybe use `packetCaptureOptions` for the type and `options` for the variable name